app/core/browse.py

```diff
@@ -59,13 +59,14 @@ def to_search_response_family(
         family_category=str(family.family_category),
         family_date=family_published_date,
         family_last_updated_date=family_last_updated_date,
-        family_source=cast(str, organisation.name),  # FIXME: check links to source
```

Comment:
I don't entirely understand what this fixme means and thus can't ascertain whether it's now met?

Reply:
This was just leftover from partial implementation of the browsing functionality. We're now getting the source name directly from the family link in the database, so it can no longer be wrong.